data/json/bionics.json

```diff
@@ -448,6 +448,7 @@
       "CLAWS_TENTACLES",
       "SHELL",
       "SHELL2",
+      "SHELL3",
```

Comment:
this exists in the list already down at line 462, should remove that. looks like it was added later, possibly

Reply:
Done.